opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -629,6 +629,7 @@ def _translate(self, record: logging.LogRecord) -> LogRecord:
             body=body,
             resource=logger.resource,
             attributes=attributes,
+            limits=self._logger_provider._log_limits,
```

Comment:
this looks like a problem - the field is only defined on one kind of LoggerProvider, there is no guarantee that it's exactly that one that is running in production, users can provide their own.

Reply:
I want to have an option to pass loglimits directly to logger_provider. Without it, LogLimits could still read from Env as it is fixed here.
Should I address limit passing in other attempt and this PR only for reading form Env works?

Reply:
Yes if you have two independent features I would recommend different PRs such that the non-controversial one can be merged.